<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultInfoDialog</class>
 <widget class="QWidget" name="ResultInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>252</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	color : rgb(200, 0, 0)
}</string>
  </property>
  <widget class="QLabel" name="m_HeaderLabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>381</width>
     <height>80</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sorry!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_BodyLabel">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>100</y>
     <width>381</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>You Lost</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>208</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="m_BottomLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="mp_OkButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
